owlbot.py

```diff
@@ -30,16 +30,26 @@
       "multiclusteringress",
     ]
```

Comment:
I wonder about the name `dependencies` here -- do these entries represent conditional external library dependencies like `pandas` or `pyarrow` in BigQuery, for instance?  Or are these just "sub-modules" of this library?

Reply:
I renamed the term `dependency` to `submodule` in owlbot.py to avoid confusion. Please re-open this thread if more clarification is needed.